src/overlays/actors/ovl_Elf_Msg4/z_elf_msg4.h

```diff
@@ -3,14 +3,18 @@
 
 #include "global.h"
 
+#define ELFMSG4_GET_7F00(thisx) (((thisx)->params & 0x7F00) >> 8)
+#define ELFMSG4_GET_8000(thisx) ((thisx)->params & 0x8000)
+#define ELFMSG4_GET_FF(thisx) ((thisx)->params & 0xFF)
+
 struct ElfMsg4;
 
 typedef void (*ElfMsg4ActionFunc)(struct ElfMsg4*, PlayState*);
 
 typedef struct ElfMsg4 {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x4];
-    /* 0x0148 */ ElfMsg4ActionFunc actionFunc;
+    /* 0x000 */ Actor actor;
+    /* 0x144 */ s32 unk_144;
```

Comment:
```suggestion
    /* 0x144 */ Actor* unk_144;
```

Reply:
src/overlays/actors/ovl_Elf_Msg4/z_elf_msg4.c:144:27: warning: assignment to ‘Actor *’ {aka ‘struct Actor *’} from incompatible pointer type ‘ElfMsg4 *’ {aka ‘struct ElfMsg4 *’} [-Wincompatible-pointer-types]
  144 |             this->unk_144 = actorPtr;

Reply:
i will need to cast like this:
this->unk_144 = (Actor*)actorPtr;